tests/unit/assessment/test_pipelines.py

```diff
@@ -112,9 +203,32 @@ def test_pipeline_without_owners_should_have_empty_creator_name():
         )
     ]
 
-    ws = Mock()
+    ws = create_autospec(WorkspaceClient)
```

Comment:
refactor to start using `workspace_client_mock`:

https://github.com/databrickslabs/ucx/blame/main/tests/unit/assessment/test_clusters.py#L22-L58

Reply:
replaced it with workspace_client_mock